admin/services/data-access/pupil-register-v2.data.service.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const { TYPES } = require('./sql.service')
+
+const sqlService = require('./sql.service')
+
+const service = {
+  /**
+   * Get list of pupils with groups
+   * @param schoolId
+   * @returns {Promise<any>}
+   */
+  getPupilRegister: function (schoolId) {
+    const sql = `
+      SELECT
+         p.id as pupilId,
+         p.foreName,
+         p.middleNames,
+         p.lastName,
+         p.urlSlug,
+         p.dateOfBirth,
+         p.school_id,
+         p.group_id,
+         ISNULL(g.name, '-') as groupName
+      FROM
+           [mtc_admin].[pupil] p LEFT JOIN
+           [mtc_admin].[group] g ON (p.group_id = g.id)
+      WHERE p.school_id = @schoolId
+      ORDER BY p.lastName, p.foreName
```

Comment:
In my view, we should do the ordering on the admin app servers and not the database.  There is one DB server; the app layer is horizontally scalable.

Reply:
@jon-shipley removed and added admin backend sorting